scripts/publicNames.txt

```diff
@@ -0,0 +1,1452 @@
+clients
+    python
+        girder_client
+            AuthenticationError
+            DEFAULT_PAGE_LIMIT
+            GirderClient
+                DEFAULT_API_ROOT
+                DEFAULT_HOST
+                DEFAULT_HTTPS_PORT
+                DEFAULT_HTTP_PORT
+                DEFAULT_LOCALHOST_PORT
+                MAX_CHUNK_SIZE
+                addFolderUploadCallback
+                addItemUploadCallback
+                addMetadataToFolder
+                addMetadataToItem
+                authenticate
+                createCollection
+                createFolder
+                createItem
+                createResource
+                createUser
+                delete
+                downloadFile
+                downloadFileAsIterator
+                downloadFolderRecursive
+                downloadItem
+                downloadResource
+                get
+                getCollection
+                getDefaultPort
+                getDefaultScheme
+                getFile
+                getFolder
+                getFolderAccess
+                getItem
+                getResource
+                getServerAPIDescription
+                getServerVersion
+                getUser
+                inheritAccessControlRecursive
+                isFileCurrent
+                listCollection
+                listFile
+                listFolder
+                listItem
+                listResource
+                listUser
+                loadLocalMetadata
+                loadOrCreateFolder
+                loadOrCreateItem
+                patch
+                post
+                put
+                resourceLookup
+                saveLocalMetadata
+                sendRestRequest
+                session
+                setFolderAccess
+                setResourceTimestamp
+                setToken
+                transformFilename
+                upload
+                uploadFile
+                uploadFileContents
+                uploadFileToFolder
+                uploadFileToItem
+                uploadStreamToFolder
+            HttpError
+            IncorrectUploadLengthError
+            REQ_BUFFER_SIZE
+            cli
+                GirderCli
+                    sendRestRequest
+                help
+                main
+devops
+    ansible
+        roles
+            girder
+                library
+                    girder
```

Comment:
Are these python symbols part of our API surface?

Reply:
do you mean the girder ansible library or specifically the roles/girder/library symbols?

Reply:
The latter. The documented API is the YAML interface that it exposes to ansible users, just wondering whether stability of the python symbols inside the library actually matters to us.

Reply:
no i don't think it matters to anyone,  i'll remove it